app/models/kudo.rb

```diff
@@ -1,6 +1,7 @@
 class Kudo < ApplicationRecord
   belongs_to :giver, class_name: 'Employee'
   belongs_to :receiver, class_name: 'Employee'
+  belongs_to :company_value, class_name: 'CompanyValue'
```

Comment:
rails first assumes, the name of the class is CompanyValue, so no need to do that (not en error of course:)
```suggestion
  belongs_to :company_value
```